src/beams/create_beams.hpp

```diff
@@ -119,8 +119,8 @@ inline Beams CreateBeams(const BeamsInput& beams_input) {
         InterpolateToQuadraturePoints{
             beams.num_nodes_per_element, beams.num_qps_per_element, beams.shape_interp,
             beams.shape_deriv, beams.qp_jacobian, beams.node_u, beams.node_u_dot, beams.node_u_ddot,
-            beams.qp_u, beams.qp_u_prime, beams.qp_r, beams.qp_r_prime, beams.qp_u_dot,
-            beams.qp_omega, beams.qp_u_ddot, beams.qp_omega_dot}
+            beams.qp_x0, beams.qp_r0, beams.qp_u, beams.qp_u_prime, beams.qp_r, beams.qp_r_prime,
+            beams.qp_u_dot, beams.qp_omega, beams.qp_u_ddot, beams.qp_omega_dot, beams.qp_x}
```

Comment:
Not related to this PR explicitly: do we want to have more readable names for our variables in beams/state?  Short aliases in our kernels are nice for reading the math, but having them here can make it harder for me to keep track of everything.  I defer that decision (and what the names should be) to you and Faisal.

Reply:
Sounds good.  I agree for the parts that actually do the math - staying closer to the theory makes it much easier to follow/read.  For these higher-level ones, I go back and forth between preferring more human readable names and sticking to theory so it's easy to visually trace them through the code.